modules/core/src/test/java/org/apache/ignite/testsuites/CoreIgniteSpringTestSuite.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.testsuites;
+
+import org.apache.ignite.internal.processors.cache.distributed.dht.GridCacheDhtMultiBackupTest;
+import org.apache.ignite.startup.cmdline.GridCommandLineLoaderTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * Spring tests.
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    GridCommandLineLoaderTest.class,
+
+    GridCacheDhtMultiBackupTest.class
+})
+public class CoreIgniteSpringTestSuite {
```

Comment:
Can we perhaps get rid of these tiny suites, move these tests to already existing suites?

Reply:
@alamar,
My proposal is to have a single "LostAndFound" suite for such tests.
And we may (should) relocate them (from this suite) at separate issues. 
For sure, we should create a special TC task for this suite and remove it on work done.

Reply:
We already have LostAndFound suites, let's just use it.
It's called IgniteLostAndFoundTestSuite

Reply:
There is an issue. Those tests are not lost. But they are implemented in core module, but suite is in spring module. The check I've developed can't understand it. So this is a reason. Then I create suites with Core prefix, and put tests to them, then replaced tests in a suite in spring module with link to the new CoreSuite.

I don't know what is a right way to do with such tests. But moving them to the LostAndFound suite is incorrect way.

Reply:
Moving suites to core module may be not enough. ClientTestSuite consist of tests that are depends on IgniteH2Indexing, but written in core module.

Reply:
> Then we can update TC
I'll prepare suites that can be easily moved to original (related to tests) module. After this discuss what to do with others.

Reply:
You may just move these tests' invocation to IgniteCacheQuerySelfTestSuiteN/IgniteSpringTestSuite and avoid problems. Can you do that?

Reply:
Now there is no need to moving test and suites. Now check aggregates test and suites info over all modules and then checks aggregation result.